builder/bundles.go

```diff
@@ -447,6 +447,22 @@ func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *b
 		for p := range bundle.AllPackages {
 			args = append(args, p)
 		}
+
+		// Retry RPM downloads to avoid timeout failures due to slow network
```

Comment:
I prefer to have a function downloadRpms(packagerCmd, baseDir, max_retries) to encapsulate that.

Reply:
sure